components/org.wso2.carbon.identity.discovery/src/main/java/org/wso2/carbon/identity/discovery/DefaultOIDCProcessor.java

```diff
@@ -73,7 +73,7 @@ public int handleError(OIDCDiscoveryEndPointException error) {
         } else if (errorCode.equals(OIDCDiscoveryEndPointException.ERROR_CODE_JSON_EXCEPTION)) {
             log.error(OIDCDiscoveryEndPointException.ERROR_MESSAGE_JSON_EXCEPTION, error);
         } else {
-            log.error("Internal server error occured. ", error);
+            log.error("Internal server error occurred. ", error);
```

Comment:
White space after the `.`. Is it on purpose?

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1301